test_app/integration_tests/ios_mocks/LICENSE

```diff
@@ -0,0 +1,59 @@
+    assignable and sublicensable license to reproduce, distribute, publicly perform and display, make, have made, sell, import, modify and make
+    derivative works based on, and otherwise use and exploit any and all suggestions, comments, ideas, and all other types of information relating to the
+    Software which is communicated directly or indirectly by Licensee to Licensor.
+8. No Warranty by Licensor: LICENSOR IS PROVIDING THE SOFTWARE AND OPEN SOURCE SOFTWARE TO LICENSEE “AS IS” AND WITHOUT WARRANTY OF ANY KIND WHATSOEVER,
+    EXPRESS OR IMPLIED. ALL IMPLIED WARRANTIES OF NONINFRINGEMENT, MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE ARE HEREBY DISCLAIMED BY
+    LICENSOR AND EXCLUDED. LICENSOR DOES NOT WARRANT THAT SOFTWARE AND OPEN SOURCE SOFTWARE OR THE FUNCTIONS CONTAINED THEREIN WILL MEET LICENSEE’S
+    REQUIREMENTS, THAT THE OPERATION OF THE SOFTWARE AND OPEN SOURCE SOFTWARE WILL BE UNINTERRUPTED OR ERROR-FREE, OR THAT DEFECTS IN THE SOFTWARE AND
+    OPEN SOURCE SOFTWARE WILL BE CORRECTED. LICENSEE IS SOLELY RESPONSIBLE FOR THE USE AND DISTRIBUTION OF OPEN SOURCE SOFTWARE AND ASSUMES ALL RISKS
+    ASSOCIATED WITH ITS USE, INCLUDING WITH RESPECT TO THE FFMPEG OPEN SOURCE SOFTWARE LICENSE.
+9. Indemnification: Licensee shall indemnify, defend and hold harmless Licensor, its affiliates, their respective successors and their respective
+    officers, directors, employees, agents, and representatives for any claim, liability, loss or damage arising out of: (a) Licensee’s breach of any
+    term of this Agreement; and (b) any claim of allegedly unauthorized use or violation of any third-party patent, copyright, trade secret, or other
+    intellectual property.
+10. Term: This Agreement shall be effective upon Licensee’s agreement to be bound by the terms of this Agreement and shall end upon termination of this
+    Agreement in accordance with the provisions set out herein. Unless otherwise authorized by Licensor, Licensee must destroy all copies and component
+    parts of the Software licensed under this Agreement within one (2) weeks of the termination of this Agreement, and Licensee may be required to
+    provide proof of such destruction to Licensor. Upon the termination of this Agreement the license shall immediately terminate and Licensee shall
+    promptly stop all use of the Software.
+11. Termination: Licensor reserves the right to terminate this Agreement on thirty (30) days' notice to Licensee for any reason whatsoever.
+12. This Agreement shall not create or be construed as creating a joint venture, co-ownership, partnership, or agency relationship between Licensee and
+    Licensor. Neither Party will have authority to or will hold itself out as having any authority to incur, assume, or create, orally or in writing, any
+    liability, obligation or undertaking of any kind in the name of, or on behalf of, or in any way binding upon, the other.
```

Comment:
I do not think this file should be here.

Reply:
Okay, I will remove both license files.